compiler/src/iree/compiler/Codegen/Common/GPU/WorkgroupReordering.cpp

```diff
@@ -229,7 +361,18 @@ struct ReorderWorkgroupsPass final
       llvm::dbgs() << "\n\n";
     });
 
-    if (failed(reorderWorkgroupsInFunc(funcOp, reorderingStrategy, logTile))) {
+    uint32_t numXCDs = 1;
+    if (IREE::GPU::TargetAttr attr = getGPUTargetAttr(funcOp))
+      if (IREE::GPU::TargetChipAttr chipAttr = attr.getChip())
+        numXCDs = chipAttr.getChipletCount();
```

Comment:
nit: IREE style is to always include braces on multi-line nesting
```suggestion
    if (IREE::GPU::TargetAttr attr = getGPUTargetAttr(funcOp)) {
      if (IREE::GPU::TargetChipAttr chipAttr = attr.getChip()) {
        numXCDs = chipAttr.getChipletCount();
      }
    }
```